src/main.js

```diff
@@ -167,6 +167,7 @@ backMainButton.addEventListener("click", showHomeView)
 //to save poster if it is unique
 saveButton.addEventListener("click", avoidDuplicates);
 
+saveView.addEventListener("dblclick", deleteOnClick);
```

Comment:
What does this delete on [double] click? The event handler function name does not provide much context on its own.